### DISCUSSION TITLE
Wallet Connect project approval pending

### DISCUSSION THREAD

**Discussion by pawanlapits** [WalletConnect/walletconnect-monorepo#1958] [Developer Support]
Hello Everyone

I have created a project 9 days ago which is 1st Feb,2023 but still not approved. any idea, why it is not approving.

**Comment by GrahamMcBain** [comment]
Hey @pawanlapits 

We work to approve as fast as possible. Apologies it's taking so long!

**Reply by pawanlapits** [reply]
@GrahamMcBain  do you need any details related to my project e.g. project ID, account email etc

**Reply by pawanlapits** [reply]
account email :  tech@zeptagram.com

**Reply by pawanlapits** [reply]
Hi @GrahamMcBain  its almost a month but wallet project still not approved. If something wrong with information. please let us know.

**Reply by pawanlapits** [reply]
also @GrahamMcBain @janus @julien51 @bpierre  please let me know where I can connect with support team.
